src/main/java/org/broadinstitute/hellbender/utils/variant/writers/HomRefBlock.java

```diff
@@ -124,6 +128,15 @@ public void add(final int pos, final int newEnd, final Genotype genotype) {
                 }
             }
         }
+        if (minPPs != null) {
+            minGQ = GATKVariantContextUtils.calculateGQFromPLs(minPPs);
+        } else if (minPLs != null) {
+            minGQ = GATKVariantContextUtils.calculateGQFromPLs(minPLs);
+        } else if (minGQ == -1) {
+            minGQ = genotype.getGQ();
```

Comment:
I would compact the two last elses as minGQ = minGQ == -1 ? genotype.getGQ() : Math.min(...);